json-webhook/functions/event-to-webhook.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch');
+
+exports.handler = async function(context, event, callback) {
+  try {
+    // IFTTT only pulls the fields value1, value2, and value3 from webhook JSON;
+    event.value1 = event.MessageSid;
+    event.value2 = event.From;
+    event.value3 = event.Body;
+
+    const res = await fetch(context.WEBHOOK_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(event),
+    });
+
+    if (res.ok) {
+      callback(null, {});
```

Comment:
maybe we could send back a twiml message about message successfully forwarded or seomthing?